### ISSUE TITLE
erzsebet.pl/profile.php fails with 500 and locks account into limbo

### ISSUE THREAD

**Issue by Ore4444** [torrentpier/torrentpier#495]
**Describe the bug**
Fail to register a new user.
No visible errors on register form.
on submit, server immediately returns 500 (request/response details are below).
username and email do register but confirmation email is not sent, so it is impossible to activate the account.
when trying to login anyway, there is error "The username you submitted is incorrect or invalid, or the password is invalid." and login fails.
when trying to forget password, there is error "Sorry, but your password cannot be retrieved because your account is currently inactive".

Basically it makes the username+email stuck in limbo. 

Is there a way to re-send the activation email?
How can I send you my email address so that you can unlock it?

---

```
http://erzsebet.pl/profile.php

POST /profile.php HTTP/1.1
Host: erzsebet.pl
Connection: keep-alive
Content-Length: 1560
Cache-Control: max-age=0
Upgrade-Insecure-Requests: 1
Origin: http://erzsebet.pl
Content-Type: multipart/form-data; boundary=----WebKitFormBoundaryJpUA2AtIXlaBq4cI
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.150 Safari/537.36 Edg/88.0.705.63
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9
Referer: http://erzsebet.pl/profile.php
Accept-Encoding: gzip, deflate
Accept-Language: en-US,en;q=0.9,he;q=0.8
Cookie: __cfduid=dbdd575ed3df236d4e7cdac950df0315c1612834340; VisitWeb_Clickunder=1612834344236

HTTP/1.1 500 Internal Server Error
Date: Tue, 09 Feb 2021 01:40:05 GMT
Content-Type: text/html; charset=UTF-8
Transfer-Encoding: chunked
Connection: keep-alive
X-Frame-Options: SAMEORIGIN
CF-Cache-Status: DYNAMIC
cf-request-id: 08260b2da80000ee7dd10ba000000001
Report-To: {"group":"cf-nel","endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report?s=fWPt4%2FuDpJ9LL5KTiv6VJpCJRrwDAFQATRNHxwsdUhWtIHd%2FoLiyJCEtA9ax%2B4G49xehU5Qi6Zc1h8YW6x0QhDUmfb%2FrjDRLTJj3nA%3D%3D"}],"max_age":604800}
NEL: {"report_to":"cf-nel","max_age":604800}
Server: cloudflare
CF-RAY: 61e9e15c4e26ee7d-CDG

mode: register
reg_agreed: 1
user_timezone: 2
username: *****
user_email: *****
new_pass: *****
cfm_pass: *****
g-recaptcha-response: 03AGdBq24D1b-r4I1YDC19i_OmwqbDauG_PpSXsylXluU4Tw8ThBzspAy0KNZv_UyDm42ATr2SQB2QpbMSOQrbKRosfGw0mKns_KOsr8vfoxc4V_ts_qksraBWr51x5dyT2kCssNgzldo0rhzW2D1DYxOW38-B-0snPOSKkQJm0OGHBYQy1kXUByLFg5oXc-lu30HOGB3XtvyxMvkEBtMU9X-BsMnduofpxfuzyC-QU8DU8S4UF6jHMeT65QFR5CALZNGP0Niah7tVuntHDZNBTroEnALvHR4kOBekUqdPAROI3TeOtkOrRFdcCx_7BcFg6bZR5bn1k8iNQrJkAsSsY3dMO0EzNRc0YkIGY51oCZtjd_Q_9A07GEGzD6oNGIEyyDk7gr4qRbtQb81-cwe8lFdlOYCAJqoViRMWKcQkLVrSMDp9dXQu_xNNfjSn5uzI_sEI79f797z8
user_lang: en
submit: Submit
```
--

**Comment by ownserver** [comment]
I have the same problem...Installation is OK but register a new user is impossible.

**Comment by belomaxorka** [comment]
Has been fixed in v2.3.0.2

https://github.com/torrentpier/torrentpier/releases/tag/v2.3.0.2
